File: airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/DefaultAirbyteMessageBufferedWriterFactory.java
/*
 * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.workers.internal;

import java.io.BufferedWriter;

public class DefaultAirbyteMessageBufferedWriterFactory implements AirbyteMessageBufferedWriterFactory {

  @Override
  public AirbyteMessageBufferedWriter createWriter(BufferedWriter writer) {
    return new DefaultAirbyteMessageBufferedWriter(writer);
  }

}

File: airbyte-cdk/java/airbyte-cdk/dependencies/src/testFixtures/kotlin/io/airbyte/workers/internal/DefaultAirbyteMessageBufferedWriterFactory.kt
/*
 * Copyright (c) 2026 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.workers.internal

import java.io.BufferedWriter

class DefaultAirbyteMessageBufferedWriterFactory : AirbyteMessageBufferedWriterFactory {
    override fun createWriter(writer: BufferedWriter): AirbyteMessageBufferedWriter {
        return DefaultAirbyteMessageBufferedWriter(writer)
    }
}
